python/doc/theory/data_analysis/smirnov_test.rst

```diff
@@ -1,52 +1,63 @@
 .. _smirnov_test:
 
-Kolmogorov-Smirnov two samples test
------------------------------------
-
-Let :math:`X` be a scalar uncertain variable modeled as a random
-variable. This method deals with the construction of a dataset prior to
-the choice of a probability distribution for :math:`X`. This statatistical
-test is used to compare two samples :math:`\left\{x_1,\ldots,x_N \right\}`
-and :math:`\left\{x'_1,\ldots,x'_M \right\}`; the goal is to determine
-whether these two samples come from the same probability distribution or
-not. If this is the case, the two samples should be aggregated in order
-to increase the robustness of further statistical analysis.
-
-The test relies on the maximum distance between the cumulative distribution
-functions :math:`\widehat{F}_N` and :math:`\widehat{F}'_M` of the samples
-:math:`\left\{x_1,\ldots,x_N \right\}` and :math:`\left\{x'_1,\ldots,x'_M \right\}`.
-This distance is expressed as follows:
+Two-sample Kolmogorov-Smirnov test
+----------------------------------
+
+The Kolmogorov-Smirnov test is a statistical test that can be used
+to test whether two given samples of data are drawn from the same
+distribution which is of dimension 1 and continuous.
+
+We denote by :math:`\left\{ x_1,\ldots,x_{\sampleSize} \right\}`
+and :math:`\left\{ y_1,\ldots,y_{\sampleSize} \right\}` two given samples of dimension 1.
```

Comment:
Shouldn't the samples be ordered? Otherwise the formula for d cannot be correct, can it?

(Again, this may hold for several files.)

Reply:
No, no need to order the sample here. The statistics calculates the difference between two empirical cdf. The formule is bady written. I modified it.